unit-tests/src/org/commcare/android/tests/initialization/AppInitializationTest.java

```diff
@@ -0,0 +1,62 @@
+package org.commcare.android.tests.initialization;
+
+import org.commcare.android.CommCareTestRunner;
+import org.commcare.android.database.DbUtil;
+import org.commcare.android.util.TestAppInstaller;
+import org.commcare.android.util.LivePrototypeFactory;
+import org.commcare.dalvik.BuildConfig;
+import org.commcare.dalvik.application.CommCareApplication;
+import org.commcare.suite.model.Profile;
+import org.javarosa.core.reference.ReferenceManager;
+import org.javarosa.core.reference.ResourceReferenceFactory;
+import org.javarosa.core.util.externalizable.PrototypeFactory;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.Robolectric;
+import org.robolectric.annotation.Config;
+
+/**
+ * Tests that use the ability to install a CommCare app and login as a test
+ * user.
+ *
+ * @author Phillip Mates (pmates@dimagi.com).
+ */
+@Config(application = org.commcare.dalvik.application.CommCareApplication.class,
+        constants = BuildConfig.class)
+@RunWith(CommCareTestRunner.class)
```

Comment:
I was thinking the prototype factory and stuff, but can refactor when it's needed (IE: when we need to expand the test set) rather than pre-emptively.